presto-main/src/test/java/com/facebook/presto/operator/aggregation/fixedhistogram/TestFixedDoubleHistogram.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.fixedhistogram;
+
+import com.google.common.collect.Streams;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.stream.IntStream;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
+public class TestFixedDoubleHistogram
+{
+    @Test
+    public void testGetters()
+    {
+        FixedDoubleHistogram histogram = new FixedDoubleHistogram(200, 3.0, 4.0);
+
+        assertEquals(histogram.getBucketCount(), 200);
+        assertEquals(histogram.getMin(), 3.0);
+        assertEquals(histogram.getMax(), 4.0);
+    }
+
+    @Test
+    public void testIllegalBucketCount()
+    {
+        try {
+            new FixedDoubleHistogram(-200, 3.0, 4.0);
+            fail("exception expected");
+        }
+        catch (IllegalArgumentException e) {
+            assertTrue(e.getMessage().contains("bucketCount"));
+        }
+    }
+
+    @Test
+    public void testIllegalMinMax()
+    {
+        try {
+            new FixedDoubleHistogram(-200, 3.0, 3.0);
+            fail("exception expected");
+        }
+        catch (IllegalArgumentException e) {
+            assertTrue(e.getMessage().contains("bucketCount"));
```

Comment:
nit: any chance to make the message assertion more specific? :)

Reply:
The assertions (in `FixedHistogramUtils`) are pretty specific, but I think the unittests should be more loose. WDYT?

Reply:
@atavory : 
>  but I think the unittests should be more loose

Not necessary, I would recommend reading other tests and following Presto coding/test convention. 

One reason is about readability, when reading this tests and see `.contains("bucketCount")` , I don't quite understand what's really expected for this exception.